app/routes/swagger/schemas.js

```diff
@@ -11,7 +11,7 @@ const schemas = {};
 const ID_FIELDS = [
   'germlineReportId', 'user_id', 'owner_id', 'createdBy_id', 'addedBy_id', 'variantId',
   'gene1Id', 'gene2Id', 'reviewerId', 'biofxAssignedId', 'logoId', 'headerId', 'templateId',
-  'product_id', 'addedById',
+  'product_id', 'addedById', 'templateId',
```

Comment:
it looks like templateId is in this list already?

Reply:
Good catch. I'll remove the duplicate.